lib/src/components/NxThreatIndicatorLegend/NxThreatIndicatorLegend.scss

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+
+.nx-threat-indicator-legend {
+  $max-grid-columns: 6;
+
+  align-items: center;
+  display: inline-grid;
```

Comment:
Inline? I would have assumed this was intended as a block element

Reply:
> Inline? I would have assumed this was intended as a block element

Changed to block. 7865898a485adc6fbd4c6459d66b0c326c22f8c2